.github/workflows/book.yml

```diff
@@ -27,6 +27,6 @@ jobs:
     - name: Build the project
       run: cargo build --verbose --examples
     - name: Build the book
-      run: mdbook build book
+      run: mdbook build docs
     - name: Run book tests
-      run: SCYLLA_URI=scylladb:9042 mdbook test -L target/debug/deps book
+      run: run: SCYLLA_URI=scylladb:9042 mdbook test -L target/debug/deps docs
```

Comment:
"run" is added twice, it should be:

``run: SCYLLA_URI=scylladb:9042 mdbook test -L target/debug/deps docs``